packages/babel-plugin-transform-react-display-name/src/index.ts

```diff
@@ -2,17 +2,24 @@ import { declare } from "@babel/helper-plugin-utils";
 import path from "path";
 import { types as t } from "@babel/core";
 
+type ReactCreateClassCall = t.CallExpression & {
+  arguments: [t.ObjectExpression];
+};
+
 export default declare(api => {
   api.assertVersion(7);
 
-  function addDisplayName(id, call) {
+  function addDisplayName(id: string, call: ReactCreateClassCall) {
     const props = call.arguments[0].properties;
     let safe = true;
 
     for (let i = 0; i < props.length; i++) {
       const prop = props[i];
+      if (t.isSpreadElement(prop)) {
+        continue;
+      }
```

Comment:
Is this just to help inference, or is it a bugfix? (If it's a bugfix, please add a test).